lib/common/auth.ts

```diff
@@ -3,12 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0.
  */
 
+/**
+ * TestCommon
```

Comment:
Question: Do we need to fill this in?

Reply:
Yeah that was leftover from the experimentation phase.